sui-execution/latest/sui-move-natives/src/crypto/poseidon.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Originally I thought it should be under `hash.rs` (where all of the other Sui added hash functions live) but I'm ok this special hash function to live on its own. @mystenmark wdyt?

Reply:
That makes sense. I'll move it to sui::hash.

Reply:
The layout of sui-move-natives should match their Move package layout

Reply:
Got it. But this is true for all the files in the crypto folder, right? If so, we should probably make a PR which fixes this.